### ISSUE TITLE
[Bug] install-gui.sh failing on Ubuntu 20.04

### ISSUE THREAD

**Issue by Jahorse** [Chia-Network/chia-blockchain#9727]
### What happened?

I tried running the `install-gui.sh` script from a terminal after activating my python virtual environment and it failed on the line `sudo apt-get install -y npm nodejs libxss1`.

I tried to research the error, but I couldn't find anything helpful. Trying to install npm and nodejs isn't necessary since I already have npm and node installed through nvm, the Node Version Manager, so I think checking if they're already available would be a suitable solution.

I don't think it's recommended to install these programs through apt-get because the versions there are usually a bit old. For instance, the version of nodejs in apt-get is v14.18.2 while v16.x.x has been active LTS since October.

### Version

1.2.12.dev124 (built from 13ff748)

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

```shell
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help resolve the situation:

The following packages have unmet dependencies:
 nodejs : Conflicts: npm
 npm : Depends: node-gyp (>= 3.6.2~) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by Jahorse** [comment]
I've put up a PR of the code change I made to be able to successfully run `install-gui.sh`.

**Comment by Jahorse** [comment]
Still waiting for my PR to be reviewed.

**Comment by ChiaMineJP** [comment]
https://github.com/Chia-Network/chia-blockchain/pull/9921
In the above PR, we overhauled `install.sh` as well as `install-gui.sh`.
You will have working installers in the next release.
Thank you for your patience.

**Comment by ChiaMineJP** [comment]
Fixed in 1.3.
